app/views/vendors/index.html.erb

```diff
@@ -1,9 +1,31 @@
 <h1>Vendors</h1>
 
 <div id="vendors">
-  <% @vendors.each do |vendor| %>
-    <p>
-      <%= link_to vendor.folio_id, vendor_url(vendor.folio_id) %>
-    </p>
-  <% end %>
+  <div>
```

Comment:
Do we need this inner `<div>` or can the containing `<div>` hold the table?

Reply:
good catch - i was flailing when i put that in.